### ISSUE TITLE
Enhance Module

### ISSUE THREAD

**Issue by HarshKhandeparkar** [publiclab/image-sequencer#1769]
![img](https://upload.wikimedia.org/wikipedia/commons/thumb/e/e6/Usm-unsharp-mask.png/220px-Usm-unsharp-mask.png)

ref: https://en.wikipedia.org/wiki/Edge_enhancement

**Comment by jywarren** [comment]
Looks cool! Maybe we name it "edge enhance"?

Also makes me want to do a local contrast module!

**Comment by HarshKhandeparkar** [comment]
Absolutely! Trying to enhance text is very difficult without a local contrast.

**Comment by HarshKhandeparkar** [comment]
After enhancing, if we reduce contrast and optionally increase exposure, text (especially handwritten) becomes very clear...
